api/v1alpha1/openapi.yaml

```
@@ -24,11 +24,26 @@ tags:
     description: Operations on EnrollmentRequest resources.
   - name: fleet
     description: Operations on Fleet resources.
+  - name: information
+    description: Operations for receiving service information.
   - name: repository
     description: Operations on Repository resources.
   - name: resourcesync
     description: Operations on ResourceSync resources.
 paths:
+  /api/v1/info/version:
```

```suggestion
  /api/version:
```
v1 is not needed because we're returning the version info
info is not needed - there is nothing else in that "directory".

---

done